pmlc/dialect/pxa/transforms/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@
     nest_loops.cc
     normalize.cc
     resize_tmps.cc
+    simplify_with_constraints.cc
```

Comment:
Hmm, looks like we need at least basic CI that calls cmake to prevent this kind of omission.